apps/gateway-ui/src/GatewayApi.ts

```diff
@@ -117,13 +117,13 @@ export class GatewayApi {
 
   requestWithdrawal = async (
     federationId: string,
-    amountSat: number,
+    msatAmountOrAll: number | string, // you can pass msat amount or 'all'
```

Comment:
This currently fails because fedimint expects `All`, not `all`. Trying to fix this deserialization in https://github.com/fedimint/fedimint/pull/3755 so we can have more consistent snake_case API.